lib/scrum/priority_name.rb

```diff
@@ -0,0 +1,15 @@
+module Scrum
+  class PriorityName
+    PRIORITY_REGEX      = /^(?:\([\d.]+\) )?P(\d+): /
+
+    def priority(name)
+      return unless m = name.match(PRIORITY_REGEX)
+      m.captures.first.to_i
+    end
+
+    def build(name, n)
```

Comment:
I now saw that there is a test for this. Actually the "P8: (3)" format is also valid. Ignore this one.